src/helpers.js

```diff
@@ -59,7 +59,10 @@ function nodePattern(opts) {
     pattern += escapeIdentifier(opts.identifier);
 
   if (opts.label)
-    pattern += ':' + escapeIdentifier(opts.label);
+    if (Array.isArray(opts.label))
+      pattern += ':' + opts.label.map(label => escapeIdentifier(label)).join(':');
```

Comment:
Can you drop the ES6 arrow function here please? The library has only used ES5 for now to avoid a transpilation step and be able to run on older engines.